backends/tfhe-cuda-backend/cuda/include/integer/integer.h

```diff
@@ -218,15 +218,17 @@ void scratch_cuda_integer_radix_comparison_kb_64(
 
 void cuda_comparison_integer_radix_ciphertext_kb_64(
     void *const *streams, uint32_t const *gpu_indexes, uint32_t gpu_count,
-    void *lwe_array_out, void const *lwe_array_1, void const *lwe_array_2,
-    int8_t *mem_ptr, void *const *bsks, void *const *ksks,
-    uint32_t lwe_ciphertext_count);
+    CudaRadixCiphertextFFI *lwe_array_out,
+    CudaRadixCiphertextFFI const *lwe_array_1,
+    CudaRadixCiphertextFFI const *lwe_array_2, int8_t *mem_ptr,
+    void *const *bsks, void *const *ksks);
```

Comment:
looks like we already had it before but do we need `void *const *bsks` or `void *const *ksks` does not it mean that data of those arrays can be modified.

Reply:
Yes but I don't see how to avoid this unfortunately, we pass a pointer to pointers in this case for multi-gpu computations.

Reply:
maybe const void *const *bsks

Reply:
I'll open an issue to try it out, not sure I can pass a `*const *const c_void` pointer from the Rust side.

Reply:
Issue open here: https://github.com/zama-ai/tfhe-rs-internal/issues/934